JustSaying.AwsTools/QueueCreation/AmazonQueueCreator.cs

```diff
@@ -6,27 +5,49 @@ namespace JustSaying.AwsTools.QueueCreation
 {
     public class AmazonQueueCreator : IVerifyAmazonQueues
     {
+        private readonly IAwsClientFactoryProxy awsClientFactory;
+        private readonly IRegionResourceCache<SqsQueueByName> queueCache = new RegionResourceCache<SqsQueueByName>();

```

Comment:
Private members should start with '_'. i.e. _awsClientFactory, _queueCache etc...